src/components/molecules/Metadata/Metadata.js

```diff
@@ -41,11 +41,29 @@ class Metadata extends LitElement {
         const key = entry.split('.')[1]; // get key name
         const title = properties[key].title;
         const value = getValueFrom(metadataList, entry);
+        const type = properties[key].type[0];
+        const format = properties[key].format;
+        const titleHtml = html`<dt class='buv-c-metadata-list__title'>${title}</dt>`;
+        let valueHtml = html`<dd class='buv-c-metadata-list__detail'>
+            <pre class='buv-c-metadata-list__detail-text'>${value}</pre></dd>`; // Standard html formatting
+        if (type === 'string') {
+          switch (format) {
+            case 'uri':
+              valueHtml = html`<dd class='buv-c-metadata-list__detail'><pre><a class='buv-c-metadata-list__detail-text'
+              href="${value}" target=”_blank”>${value}</a></pre></dd>`;
+              break;
+            case 'email':
+              valueHtml = html`<dd class='buv-c-metadata-list__detail'><pre><a class='buv-c-metadata-list__detail-text'
+                href="mailto:${value}">${value}</a></pre></dd>`;
+              break;
+            case 'phoneNumber':
+              valueHtml = html`<dd class='buv-c-metadata-list__detail'><pre><a class='buv-c-metadata-list__detail-text'
+                href="tel:${value}">${value}</a></pre></dd>`;
```

Comment:
I think this could be simplified, all that changes in this code compared to the current one is wrapping the value in a link tag. 
Additionally, if you think about it the only variation then is really about the `href` value. 
We are left with the special case of the `target=_blank`, which could be handled by a boolean flag.

In terms of readability, it would also be better to extract this logic into a standalone function, which you could also test with jest.

Otherwise I think it's a good addition.

Do you mind trying to simplify and test this code? I'll handle the cherry-picking and opening a mergeable PR.

Thank you

Reply:
Jest is not too ideal for testing `lit-html`, but you can manage some things. It's a bit consequential, and thus a bit weak (it will not entirely test the truth you want), but it's better than nothing and will also be more efficient and easier to write than the browser tests.

You can take a look at this for reference: https://github.com/blockchain-certificates/blockcerts-verifier/pull/146/files#diff-2e3de4ff0f062e67bd23d43f213b6114

Please note that apparently it's not possible to test a class extending `LitElement`, at least I had trouble when I tried, that's why I'm suggesting a standalone function to test your code.

Reply:
Goodmorning,
I performed a refactoring of this feature.
I tried to export logic in a pure standalone function, but I had problems due to the 'html' component of lit-element. 
I finally added a new 'atom' component for rendering the links, and I wrote unit tests
for this. The drawback is that for testing a needed to stringify the lit-element-html object,
so I performed assertion based on this.
I believe this can be a good compromise,
let me know what do you think.
Thank you

Emiliano